### PR TITLE
add info about squishymedia

### PR DESCRIPTION
**Pull request by sappelhoff**
related to discussion on BIDS governance, see: https://docs.google.com/document/d/1R-J2lL9V_wIkYhye4zH-feyl4P4J8NyO40rIYyY141o/edit?disco=AAAADI0g6l4

cc @effigies @franklin-feingold @yarikoptic 

Please advise on the wording.

### CODE REVIEW

**Comment by effigies** [comment]
Russ' name is spelled wrong, but this otherwise seems fine.

**Review by yarikoptic — CHANGES_REQUESTED** [review]
I more often hear use of "supported" as in money in such context, so I would have used "done" or alike. Also a dedicated Acknowledgments section would be more appropriate place.

File: bids-validator/README.md [hunk 1]
```diff
@@ -266,6 +266,11 @@ False
 
 ## Development
 
+The development of `bids-validator` is currently supported by
+[squishymedia](https://squishymedia.com/), who are in turn financed by a grant
```
Comment (effigies) [inline bids-validator/README.md]:
```suggestion
[Squishymedia](https://squishymedia.com/), who are in turn financed by a grant
```

**Comment by sappelhoff** [comment]
After discussion on Slack, we agreed that it's probably a good idea to ask each PI with a grant for BIDS-related activities whether they want to be listed here.

@yarikoptic @satra 

@effigies perhaps you can tag some other people that you know of.

@franklin-feingold the formatting of Russ' grants is as you told me over Slack. But perhaps you could provide the more formal version of it?

Currently it's looking like this:

```
- BIDS R24,
- OpenNeuro R24
- NSF Spokes grant
```

**Comment by franklin-feingold** [comment]
the more formal/detailed version of our grants that I know of supporting squishy:
- [Brain Imaging Data Structure (BIDS) Derivatives grant funded by the NIMH: R24MH114705](http://grantome.com/grant/NIH/R24-MH114705-01) (PI: Poldrack)
- [OpenNeuro grant funded by the NIMH: R24MH117179](http://grantome.com/grant/NIH/R24-MH117179-01) (PI: Poldrack)
- [Spokes: MEDIUM: WEST grant funded by the NSF: 1760950](http://grantome.com/grant/NSF/IIS-1760950) (PI: Poldrack & Gorgolewski)

**Comment by yarikoptic** [comment]
So it could look like
```markdown
Development and contributions were supported through following federally funded projects/grants
- [BIDS Derivatives (NIMH: R24MH114705, PI: Poldrack)](http://grantome.com/grant/NIH/R24-MH114705-01)
- ...
```
rendered as

Development and contributions were supported through following federally funded projects/grants
- [BIDS Derivatives (NIMH: R24MH114705, PI: Poldrack)](http://grantome.com/grant/NIH/R24-MH114705-01)
- ...

and so on.  You could add

```markdown
- [ReproNim](http://repronim.org) ([NIH-NIBIB P41 EB019936, PI: Kennedy](https://projectreporter.nih.gov/project_info_description.cfm?aid=8999833))
```

**Comment by sappelhoff** [comment]
great, thanks for your help @yarikoptic and @franklin-feingold! Ready for a review

**Review by franklin-feingold — APPROVED** [review]

**Review by effigies — APPROVED** [review]
LGTM. I put this below the new publish section from #797.

**Pull request merged**